README.md

```diff
@@ -127,7 +127,7 @@ In order to receive webmentions (this is optional, you may also use the plugin f
 
 Per the specification ([ch 3.2](https://www.w3.org/TR/webmention/#receiving-webmentions)), incoming webmentions are always placed in a backlog queue for asynchronous processing (this is to mitigate the risk of DDoS attacks by flooding your site with webmentions). In order to have these webmentions processed, this queue needs to be run regularly.
 
-First, set a secret key with at least 10 characters in your `site/config/config.php` (the key may NOT include any of the following: `&` `%` `#` `+` nor a space sign ` `):
+First, set a secret key with at least 10 characters in your `site/config/config.php` (the key may NOT include any of the following: `&` `%` `#` `+` nor a space sign ``):
```

Comment:
Not quite sure what happened here, but for example the space sign ` ` got deleted. Some automated formatting going rogue?